ArduPlane/quadplane.h

```diff
@@ -347,6 +347,7 @@ class QuadPlane
     
     // timer start for transition
     uint32_t transition_start_ms;
+    AP_Float transition_initial_pitch;
```

Comment:
AP_Float should be "float" unless you want this to be a new parameter

Reply:
fixed